app/views/bookings/_my_spaces.html.erb

```diff
@@ -0,0 +1,51 @@
+
+<% user = @bookings.first.user %>
+<% spaces = user.spaces  %>
+<div class="container pl-0">
+  <% user.spaces.each do  |space| %>
+    <div class="container pt-2 mb-3 pb-4" style="background-color: white;">
+      <div class="card mb-1 mt-3" style="flex-direction: row; padding: 10px;  border:none;" >
+        <%  if space.photo %>
+         <%= cl_image_tag(space.photo, width: 160, height: 160, crop: :fill) %>
+        <% end  %>
+
+        <div class="card-body " >
+          <h5 class="card-title" style="margin: 0; padding: 0;"><%= space.address %></h5>
+          <h5 class="card-text" style="margin: 0; padding: 0;">Price: <%= space.price %> euro</h5>
+          <h5 class="card-text mb-2" style="margin: 0; padding: 0;">Size: <%= space.size %> m^2</h5>
+          <%= link_to "Edit", space_path(space), class: "button h5" %>
+        </div>
+      </div>
+      <div class="container">
+        <% space.bookings.each do |booking| %>
+          <div class="card" style="padding: 0px">
+            <% case booking.status
+              when "confirmed"
+                headerClass = "bg-success"
+              when "pending"
+                headerClass = "bg-warning"
+              when "reject"
+                headerClass = "bg-danger"
+              else
+                headerClass = "bg-warning"
```

Comment:
why warning? Canno be it just white or grey? because warning shows that there was a book-request.